src/NetPanzer/meson.build

```diff
@@ -4,7 +4,7 @@ src_basename = [
   'package.cpp'
 ]
 foreach file : src_basename
-  src += [join_paths('../Lib', file)]
+  src += [join_paths(meson.source_root(), 'src/Lib', file)]
```

Comment:
these changes were required to get it to build

Reply:
Using [files()](https://mesonbuild.com/Reference-manual_functions.html#files) would probably be better, and I would have done that originally if I'd thought of it.